lib/source/consul.js

```diff
@@ -24,9 +24,9 @@ class Consul extends Source.Polling(Parser) { // eslint-disable-line new-cap
    */
   constructor(name, opts) {
     const options = Object.assign({
-      host: Consul.DEFAULT_ADDRESS,
-      port: Consul.DEFAULT_PORT,
-      secure: false
+      host: Config.get('consul:host'),
+      port: Config.get('consul:port'),
+      secure: Config.get('consul:secure')
     }, opts);
```

Comment:
If you look in the [default config](https://github.com/rapid7/propsd/blob/781799eec8343f124dea0f7323fbb365ebac8fd5/config/defaults.json#L15-L19) we already specify some Consul options. Imo better to not duplicate these things.